### ISSUE TITLE
Courseplay in Multiplayer Setting

### ISSUE THREAD

**Issue by akuenzi** [Courseplay/courseplay#2856]
We are having difficulty using Courseplay on our multiplayer server.  On single player everything seems to work perfectly, beyond the occasional hiccup.  In multiplayer, it's been frustrating.

Sometimes my mouse pointer disappears.  Sometimes I click on the 'eye' to show the route, and it doesn't draw it on the screen -- the hud may disappear, but no route gets drawn, and errors pelt the log.  Here of late when I select a course for my tractor to run, it grabs TWO courses and gets all confused.  Sometimes I grab a course, and when I plot it on the screen, the tractor refuses to follow what was plotted, but is rather offset a couple meters... and I never told it to be offset, and the offset shows as 0.  Other times when I ask it to grab the 'fertilizing' course, it instead grabs the 'windrow' course.

And then yet at other times, it functions perfectly as one would expect.

I don't know if this makes any difference, but I typically access our server from two locations.  One on my work computer, and one on my home computer.  Courses have been generated in both places.  The most typical vehicle to run a course has been a modded JD tractor, and one tractor that came with the game.  I sometimes use a truck to run some courses, but mainly it's these tractors, performing various tasks such as fertilizing, windrowing, picking up windrows or baling, and combi.  Lots of point-to-point running too.  The point-to-point courses seem to function okay.  It's the field stuff where things seem to break down in random ways.

My most recent log:
[log.txt](https://github.com/Courseplay/courseplay/files/2334145/log.txt)

I'm using courseplay version 5.3.0.0.

Thank you for your time.

**Comment by pops64** [comment]
Both issuses have been documented. #2829 and #2848. For the map draw on screen could you produce a log with just that error so we know what is the cause.

**Comment by pops64** [comment]
If you use ploughs or pick up wagon for bails CP automatically sets this for all vanilla and dlc tools. There are some others.

**Comment by akuenzi** [comment]
I didn't realize CP automatically set those.  Thanks for that.  I've now tried the 'reset' approach, and that seemed to do the trick, and will provide a workaround on the issue.  But I'll keep watching on the drawing error to see if it happens again.

**Comment by pops64** [comment]
Yeah on SP the offset usually gets set back to zero. It is something I am looking to improve after I get back from vacation. The MP issuse is something different
